### PR TITLE
Calibration XRF

### PR DESCRIPTION
**Pull request by DarioZappala**
### `notebook_utils.py`
- Added function `read_parameters_calibration_xrf` to read calibration parameters from file.

### Classes `Spectra`, `SpectraXRF` and `SpectraXRD`
- Moved methods `calibrate_from_parameters()` and `calibrate_from_file()` from `SpectraXRD` to `Spectra`, to enable calibration both in XRD and XRF spectra.
- Added methods `energy()` and `energy_range()` to class `SpectraXRF`. They are analogous to method `theta()` and `theta_range()` of `SpectraXRD`.

### CODE REVIEW

File: XRDXRFutils/notebook_utils.py [hunk 1]
```diff
@@ -163,6 +163,19 @@ def shorten_string(s, length_max):
     return s_short
 
 
+def read_parameters_calibration_xrf(filename):
```
Comment (zpreisler) [inline XRDXRFutils/notebook_utils.py:166]: What are these files that we are reading?

File: XRDXRFutils/spectra.py [hunk 2]
```diff
@@ -78,6 +93,15 @@ def from_spe(self, filename):
                 warnings.warn(f'Unknown data format in file \'{filename}\'')
                 return None
 
+    @property
+    def energy(self):
+        return self.fce_calibration(self.channel, *self.opt)
```
Comment (zpreisler) [inline XRDXRFutils/spectra.py:101]: Isnt opt part of Calibration class?
I do not see opt defined in Spectra nor SpectraXRF

File: XRDXRFutils/notebook_utils.py (hunk 1 above)
Comment (DarioZappala) [reply XRDXRFutils/notebook_utils.py:166]: I attach an example of these files. They are simple files that contain the parameters of calibration for XRF spectra. In particular, they contain a, b, c for the calibration of type `a x^2 + b x + c`.
Paolo and Eva need to be able to calibrate any XRF spectrum taking that file as the source of calibration.

[calib_XRF_2nd.calib.zip](https://github.com/zpreisler/XRDXRFutils/files/10881333/calib_XRF_2nd.calib.zip)

**Review by RosarioAndolina — COMMENTED** [review]
Dario resolved the conflicts with synthetic part correctly.

**Pull request merged**